src/content/app/genome-browser/hooks/useGenomeBrowser.ts

```diff
@@ -178,27 +251,68 @@ const useGenomeBrowser = () => {
     genomeBrowser.send(action);
   };
 
+  const toggleTrackName = (params: {
```

Comment:
Don't these toggle functions also belong in the useBrowserTrackConfig hook?

Reply:
✅